Unhandled exception in discovery II

This appears to be different from #2176 

```
2020-06-30 08:17:31.670+00:00 | discovery-tasks-1 | ERROR | DefaultSchedulers | Unhandled exception:
reactor.core.Exceptions$ErrorCallbackNotImplemented: java.util.NoSuchElementException: No value present
Caused by: java.util.NoSuchElementException: No value present
        at java.util.Optional.orElseThrow(Optional.java:375) ~[?:?]
        at org.ethereum.beacon.discovery.pipeline.handler.NodeIdToSession.getRemoteSocketAddress(NodeIdToSession.java:125) ~[discovery-0.3.6.jar:0.3.6]
        at org.ethereum.beacon.discovery.pipeline.handler.NodeIdToSession.getOrCreateSession(NodeIdToSession.java:93) ~[discovery-0.3.6.jar:0.3.6]
        at org.ethereum.beacon.discovery.pipeline.handler.NodeIdToSession.handle(NodeIdToSession.java:87) ~[discovery-0.3.6.jar:0.3.6]
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:189) ~[reactor-core-3.3.0.RELEASE.jar:3.3.0.RELEASE]
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:203) ~[reactor-core-3.3.0.RELEASE.jar:3.3.0.RELEASE]
        at reactor.core.publisher.FluxPeekFuseable$PeekFuseableSubscriber.onNext(FluxPeekFuseable.java:203) ~[reactor-core-3.3.0.RELEASE.jar:3.3.0.RELEASE]
        at reactor.core.publisher.FluxReplay$SizeBoundReplayBuffer.replayNormal(FluxReplay.java:811) ~[reactor-core-3.3.0.RELEASE.jar:3.3.0.RELEASE]
        at reactor.core.publisher.FluxReplay$SizeBoundReplayBuffer.replay(FluxReplay.java:895) ~[reactor-core-3.3.0.RELEASE.jar:3.3.0.RELEASE]
        at reactor.core.publisher.ReplayProcessor.onNext(ReplayProcessor.java:442) ~[reactor-core-3.3.0.RELEASE.jar:3.3.0.RELEASE]
        at reactor.core.publisher.FluxCreate$IgnoreSink.next(FluxCreate.java:618) ~[reactor-core-3.3.0.RELEASE.jar:3.3.0.RELEASE]
        at reactor.core.publisher.FluxCreate$SerializedSink.next(FluxCreate.java:153) ~[reactor-core-3.3.0.RELEASE.jar:3.3.0.RELEASE]
        at org.ethereum.beacon.discovery.pipeline.PipelineImpl.push(PipelineImpl.java:43) ~[discovery-0.3.6.jar:0.3.6]
        at org.ethereum.beacon.discovery.DiscoveryManagerImpl.executeTaskImpl(DiscoveryManagerImpl.java:151) ~[discovery-0.3.6.jar:0.3.6]
        at org.ethereum.beacon.discovery.DiscoveryManagerImpl.ping(DiscoveryManagerImpl.java:162) ~[discovery-0.3.6.jar:0.3.6]
        at org.ethereum.beacon.discovery.task.LiveCheckTasks.lambda$add$2(LiveCheckTasks.java:55) ~[discovery-0.3.6.jar:0.3.6]
        at org.ethereum.beacon.discovery.scheduler.ErrorHandlingScheduler.runAndHandleError(ErrorHandlingScheduler.java:70) ~[discovery-0.3.6.jar:0.3.6]
        at org.ethereum.beacon.discovery.scheduler.ErrorHandlingScheduler.lambda$execute$1(ErrorHandlingScheduler.java:46) ~[discovery-0.3.6.jar:0.3.6]
        at org.ethereum.beacon.discovery.scheduler.Scheduler.lambda$execute$0(Scheduler.java:35) ~[discovery-0.3.6.jar:0.3.6]
        at org.ethereum.beacon.discovery.scheduler.ExecutorScheduler.lambda$execute$0(ExecutorScheduler.java:32) ~[discovery-0.3.6.jar:0.3.6]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515) [?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) [?:?]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) [?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
        at java.lang.Thread.run(Thread.java:830) [?:?]
```

--- a/gradle/versions.gradle
+++ b/gradle/versions.gradle
@@ -130,7 +130,7 @@ dependencyManagement {
     dependency "org.testcontainers:testcontainers:1.12.3"
     dependency "org.testcontainers:junit-jupiter:1.12.3"
 
-    dependency 'tech.pegasys.discovery:discovery:0.3.6'
+    dependency 'tech.pegasys.discovery:discovery:0.3.7'
 
     dependency 'tech.pegasys.signers.internal:bls-keystore:1.0.1'
   }

